### DISCUSSION TITLE
Clarification on Security Practices

### DISCUSSION THREAD

**Discussion by londa100** [community/community#75609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is the preferred method for reporting potential security vulnerabilities? Are there specific steps to follow?

**Comment by elderguardian** [comment]
Every repository should have its own specifications about this. Just check it for a CODE_OF_CONDUCT file or issue templates

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @londa100 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
